src/SFA.DAS.Reservations.Functions.Reservations.AcceptanceTests/Features/DeleteReservation.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: Reservation deleted
+	In order to know when a reservation has been deleted
+	I want to be told when the reservation search index is updated
+
+
+Scenario: Reservation search index should be updated on reservation deletion
+	Given I have a reservation ready for deletion
+	When a delete reservation event is triggered
+	Then the reservation search index should be updated with the deleted reservation removed
+
+Scenario: Employer should be notified on reservation deletion
+	Given I have a reservation ready for creation
+	When a delete reservation event is triggered
+	Then the employer should be notified of the deleted reservation
```

Comment:
Would be good to add another scenario - this one is actually covering when a provider has deleted an employers reservation. Would also be good to cover the following:

- Employer deletes own reservation - no email should be sent
- Employer deletes reservation by provider - no email should be sent

Also look at covering Employers with different notification settings

Reply:
Done